### PR TITLE
Add drawingboard.js@0.4.6 w/ git auto-update

### LINKED ISSUE #11097: [Request] Add drawingboard.js
**Library name:** drawingboard.js
**Git repository url:** https://github.com/Leimi/drawingboard.js
**npm package name or url** (if there is one): 
**License (List them all if it's multiple):** MIT
**Official homepage:** http://leimi.github.io/drawingboard.js
**Wanna say something? Leave message here:**

### PR DESCRIPTION
**Pull request by dakshshah96**
close #11097, cc @Leimi

Pull request for issue: #11097 

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/Leimi/drawingboard.js
 * Official website (optional, not the repository): http://leimi.github.io/drawingboard.js
 * License and its reference: MIT
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 65
   - Count of stars: 1,430
   - Count of forks: 168

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/drawingboard.js/package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "drawingboard.js",
+  "version": "0.4.6",
+  "filename": "drawingboard.min.js",
+  "description": "A canvas based drawing app that you can integrate easily on your website",
+  "homepage": "http://leimi.github.io/drawingboard.js",
```
Comment (maruilian11) [inline ajax/libs/drawingboard.js/package.json]: please change to use `https`, thanks!

**Comment by dakshshah96** [comment]
@maruilian11 Done 👍

**Comment by PeterDaveHello** [comment]
@maruilian11 why don't you use review request but assign?

**Comment by PeterDaveHello** [comment]
This needs to be reviewed by @pvnr0082t first...

**Comment by dakshshah96** [comment]
@pvnr0082t Please review this 👍

**Pull request merged**